pupil-spa/src/app/check-complete/check-complete.component.html

```diff
@@ -1,19 +1,23 @@
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <header class="page-header">
-      <h1 class="heading-xlarge" tabindex="0">Thank you</h1>
-    </header>
+<app-header style="width:100%"></app-header>
```

Comment:
I am not particularly in favor of inline styles as they can be very difficult to manage and maintain.
Would you mind moving it?

Reply:
pretty sure we have a CSP policy for this too.  or at least we wanted one, but couldnt enforce it due to the amount of inline CSS 🤔